tools/stress_ng/.shed.yml

```diff
@@ -0,0 +1,11 @@
+categories:
+  - Web Services
+description: stress test a computer system in various selectable ways
+long_description: |
+  stress-ng will stress test a computer system in various selectable
+  ways. It was designed to exercise various physical subsystems of a
+  computer as well as the various operating system kernel interfaces.
+owner: bgruening
```

Comment:
Could you maybe upload this as a different owner ? My concern in the IUC applies here as well, if you just install all tools by a trusted owner like iuc or bgruening you may end up with this tool on your server without intending to.

Reply:
I can do this if you like, but honestly, this "user" is active for 10 years and has a lot of repos that should not be installed, e.g. with old tool_dependencies etc ... we still do not install all IUC tools, but offer curated tool.yaml files.

Reply:
My thinking is that missing dependencies is different from a tool that could be used for somewhat malicious operations ... but I guess in the scenario of someone just installing everything there;d be many more problems anyway. I guess it's ok.

Reply:
Oh I hope that tool can not do anything malicious. They are still running in a cgroup context and the scheduler will kill it if they use too many resources. Do I miss something?

Reply:
But it can simulate many different things that are probably not controlled for by a scheduler, like IO, network and HDD activity, which I could see impacting other jobs on shared resources.

Reply:
Yes, this is true, but not different from Blast :sob:  ... on a positive note, this is exactly what I would like to test. How can we jail our jobs better and make our cgroup measurements better. ping @natefoo ... I wanted to eitherway inform you about this when we have installed the tool.

@mvdbeek let me know if I should move this to a different user, really happy to do it.

Reply:
It's what I would do, but I agree that you could find other tools do the same thing if you were intent on it.

Reply:
Ok, done.